client/src/components/classesPage/ClassCreateView.js

```diff
@@ -363,33 +393,46 @@ function ClassCreateView(props) {
   };
 
   return (
-    <Grid className={props.classes.wrapper}>
-      {stage.onListForm ? (
-        <ListForm
-          file={file}
-          setFile={setFile}
-          recipientData={recipientData}
-          listData={listData}
-          setListData={setListData}
-          stage={stage}
-          setStage={setStage}
-          setRecipientData={setRecipientData}
-        />
-      ) : null}
-
-      {stage.onCampaignForm ? (
-        <CampaignForm
-          campaignData={campaignData}
-          setCampaignData={setCampaignData}
-          stage={stage}
-          setStage={setStage}
-          sendAllToSendgrid={sgDb}
-          // setTimeData={setTimeData}
-          timeTriData={timeTriData}
-          setTimeTriData={setTimeTriData}
+    <>
+      {isLoading && (
```

Comment:
I'll have to start using this &&, it's really clean

Reply:
Yes! It's really helpful in a lot of cases.